### ISSUE TITLE
Accessibility: audit check

### ISSUE THREAD

**Issue by rachael-t** [nickstaylor/refactor-tractor#57]
The app must score as close to 100% in an Accessibility Audit as possible. Be prepared to explain any accessibility audits the application is failing.

- Click on the ‘Audits’ panel of dev tools
- Select the following settings to run an accessibility check on the page
- Click ‘Run Audits’ (your dev tools may disappear for a while, or your screen may go blank, just wait it out)

Read through the passing, failing, non-applicable and manual audits:
- What accessibility concerns have come to light? WHY are they accessibility concerns?
- What strategies may you have learned based on the failing audits? Passing audits?
- Run the audit on your fitlit application, take note of what issues you might need to resolve
